syft/frameworks/torch/hook/hook.py

```diff
@@ -477,155 +477,6 @@ def _hook_torch_module(self):
 
                 self._perform_function_overloading(module_name, torch_module, func)
 
-    def _hook_crypten(self):
```

Comment:
@IamRavikantSingh sorry...This is needed (my bad :( ). You can move it where @youben11 suggsted. We use another load function syft than what it is used in CrypTen because we use the tags to properly load the necessary data.